sdks/python/apache_beam/examples/wordcount_it_test.py

```diff
@@ -95,7 +95,7 @@ class WordCountIT(unittest.TestCase):
   # lines read from expected output.
   DEFAULT_CHECKSUM = '33535a832b7db6d78389759577d4ff495980b9c0'
 
-  @attr('IT')
+  @attr('IT', 'Py3IT')
```

Comment:
Should we just use the same 'IT' tag, but limit the tests we run inside a suite if not all IT tests are supporting Python3 now?

Reply:
sg.